script.js

```diff
@@ -0,0 +1,521 @@
+    ['ф', 'Ф', 'a', 'A', 'KeyA'],
+    ['ы', 'Ы', 's', 'S', 'KeyS'],
+    ['в', 'В', 'd', 'D', 'KeyD'],
+    ['а', 'А', 'f', 'F', 'KeyF'],
+    ['п', 'П', 'g', 'G', 'KeyG'],
+    ['р', 'Р', 'h', 'H', 'KeyH'],
+    ['о', 'О', 'j', 'J', 'KeyJ'],
+    ['л', 'Л', 'k', 'K', 'KeyK'],
+    ['д', 'Д', 'l', 'L', 'KeyL'],
+    ['ж', 'Ж', ';', ':', 'Semicolon'],
+    ['э', 'Э', '\'', '\x22', 'Quote'],
+    ['return', 'return', 'return', 'return', 'Enter'],
+  ],
+  [
+    ['shift', 'shift', 'shift', 'shift', 'ShiftLeft'],
+    ['я', 'Я', 'z', 'Z', 'KeyZ'],
+    ['ч', 'Ч', 'x', 'X', 'KeyX'],
+    ['с', 'С', 'c', 'C', 'KeyC'],
+    ['м', 'М', 'v', 'V', 'KeyV'],
+    ['и', 'И', 'b', 'B', 'KeyB'],
+    ['т', 'Т', 'n', 'N', 'KeyN'],
+    ['ь', 'Ь', 'm', 'M', 'KeyM'],
+    ['б', 'Б', '<', ',', 'Comma'],
+    ['ю', 'Ю', '>', '.', 'Period'],
+    ['/', '?', '/', '?', 'Slash'],
+    ['shift', 'shift', 'shift', 'shift', 'ShiftRight'],
+  ],
+  [
+    ['control', 'control', 'control', 'control', 'ControlLeft'],
+    ['option', 'option', 'option', 'option', 'AltLeft'],
+    ['command', 'command', 'command', 'command', 'MetaLeft'],
+    ['\x20', '\x20', '\x20', '\x20', 'Space'],
+    ['command', 'command', 'command', 'command', 'MetaRight'],
+    ['option', 'option', 'option', 'option', 'AltRight'],
+    ['&#9666;', '&#9666;', '&#9666;', '&#9666;', 'ArrowLeft'],
+    ['&#9652;', '&#9652;', '&#9652;', '&#9652;', 'ArrowUp'],
+    ['&#9662;', '&#9662;', '&#9662;', '&#9662;', 'ArrowDown'],
+    ['&#9656;', '&#9656;', '&#9656;', '&#9656;', 'ArrowRight'],
+  ],
+];
+
+const functionalKeys = ['Backspace', 'Tab', 'Delete', 'CapsLock', 'Enter', 'ShiftLeft', 'ShiftRight', 'ControlLeft', 'AltLeft', 'MetaLeft', 'MetaRight', 'AltRight', 'ArrowLeft', 'ArrowRight', 'ArrowDown', 'ArrowUp', 'Space'];
+
+// creating wrapper
+const wrapper = document.createElement('div');
+wrapper.classList.add('wrapper');
+
+// textarea creation
+const textarea = document.createElement('textarea');
+textarea.setAttribute('rows', 20);
+textarea.setAttribute('cols', 100);
+textarea.setAttribute('tabindex', -1);
+textarea.classList.add('textarea');
+wrapper.appendChild(textarea);
+
+// creating keyboard structure
+const keyboard = document.createElement('div');
+keyboard.classList.add('keyboard');
+
+// adding rows
+for (let row = 0; row < keyboardSymbols.length; row += 1) {
+  const rowDiv = document.createElement('div');
+  rowDiv.classList.add('row');
+
+  // filling rows with keys
+  for (let key = 0; key < keyboardSymbols[row].length; key += 1) {
+    const keyDiv = document.createElement('div');
+    keyDiv.classList.add('key');
+
+    const ru = document.createElement('div');
+    ru.classList.add(keyboardSymbols[row][key][keyboardSymbols[row][key].length - 1], 'ru', 'on');
+
+    if (functionalKeys.includes(keyboardSymbols[row][key][keyboardSymbols[row][key].length - 1])) {
+      keyDiv.classList.add(keyboardSymbols[row][key][keyboardSymbols[row][key].length - 1], 'functional');
```

Comment:
looks like magic. hard to figure out what the code does. 
It is great that it works, but may be difficult to maintain. 
the same is true for ru, eng
Try to find another way , if possible.